app/src/main/java/com/infomaniak/mail/ui/main/menu/MoveFragment.kt

```diff
@@ -75,13 +79,31 @@ class MoveFragment : MenuFoldersFragment() {
         super.onViewCreated(view, savedInstanceState)
         setSystemBarsColors()
 
+        bindAlertToViewLifecycle(inputDialog)
+        setupRecyclerView()
         setupListeners()
-        setupFolderAdapters()
+        observeFolders()
         setupCreateFolderDialog()
         observeNewFolderCreation()
         observeSearchResults()
     }
 
+    private fun setupRecyclerView() = with(binding.foldersRecyclerView) {
+
+        adapter = injectedFolderAdapter(
+            isInMenuDrawer = false,
+            onFolderClicked = ::onFolderSelected,
+        )
```

Comment:
I would have probably put it on a single line, do you prefer it on two separate lines?